src/DeviceInterface/ExtensionInterface.js

```diff
@@ -111,7 +85,12 @@
         return this.hasLocalAddresses
     }
 
-    setupAutofill () {
+    async setupAutofill () {
+        /**
+         * In the extension, we must resolve `inContextSignup` data as past of setup
+         */
+        await this.inContextSignup.init();
```

Comment:
When we know of extra data that the extension needs, just fetch it inside 'setupAutofill' for each feature.